tests/data/say-formatting/litexa/main.litexa

```diff
@@ -1,14 +1,14 @@
 launch
   say "There should be
-    a space    between   
+    a space    between
     word and salad here: word {'salad'}."
-  say " There   
-    is   
-    {'1'}   
-    space     
-    {'between'}    
-    each  
-    word. 
+  say " There
+    is
+    {'1'}
+    space
+    {'between'}
+    each
+    word.
```

Comment:
The trailing spaces in these tests (and those below) were actually intentional (might have been accidentally removed in the original commit): Would re-add them (might require temporarily turning off 'remove trailing whitespaces' in your IDE if you have that setting enabled).